src/pages/calendar/components/Section.tsx

```diff
@@ -16,6 +18,33 @@ import { useDarkMode } from 'lib/hooks/useDarkMode';
 import { Schedule } from './Schedule';
 import { SeatInfo } from './Seats';
 
+type BadgeProps = {
+  name: string;
+  color: string;
+};
+
+const ruleNames: string[][] = [
+  ['Reserved for BSENG students', 'SENG Only'],
+  ['Computer Science program.', 'CSC Only'],
+  ['BEng students', 'ENGR Only'],
+  ['Faculty of Engineering', 'ENGR/CSC Only'],
+  ['SCIENCE students', 'SCI Only'],
+  ['BEng and BSEng students', 'BEng/BSeng Only'],
+  ['BME, BSEN, CENG, ELEC students', 'BME/SENG/CENG/ELEC Only'],
+];
```

Comment:
We should likely use objects here (typed) rather than an array here given it has some room for error.